### PR TITLE
Fixes due ONO API changes

### PR DESCRIPTION
**Pull request by kkateq**

### CODE REVIEW

File: src/client/components/PullRequestList/PullRequestListItem/index.js [hunk 1]
```diff
@@ -65,10 +65,10 @@ class PullRequestListItem extends Component {
                   style={{ textDecoration: 'none', color: 'rgb(59, 120, 155)' }}
                   to={{
                     pathname: buildProjectLink(pullRequest.origin.repository.fullName),
-                    query: { branch: pullRequest.origin.branch } }}
+                    query: { branch: pullRequest.origin.name } }}
```
Comment (skovhus) [inline src/client/components/PullRequestList/PullRequestListItem/index.js]: This link is only valid if the `origin.type === 'BRANCH'`. Should we handle REV, TAG and the other types here?

File: src/client/components/PullRequestSummary/PullRequestSummary.stories.js [hunk 2]
```diff
@@ -32,14 +32,14 @@ const pullRequestFixture = {
   ],
   origin: {
     url: 'unity/unity#my-pr',
-    branch: 'bar',
+    name: 'bar',
```
Comment (skovhus) [inline src/client/components/PullRequestSummary/PullRequestSummary.stories.js]: Let us add `type: 'BRANCH'`

File: src/universal/services/ono/queries/pullRequest/index.js [hunk 3]
```diff
@@ -20,13 +20,13 @@ query ($id: Int!) {
       }
     }
     origin {
-      branch
+      name
```
Comment (skovhus) [inline src/universal/services/ono/queries/pullRequest/index.js]: Let us add type here.

File: src/universal/services/ono/queries/pullrequests/index.js [hunk 4]
```diff
@@ -112,18 +105,30 @@ export type RepositoryGraphType = {
   name: string
 }
 
+export const PullRequestSource = {
+  BRANCH: 'branch',
+  REV: 'rev',
```
Comment (skovhus) [inline src/universal/services/ono/queries/pullrequests/index.js]: Is this the valid type? I think it should be BRANCH and REVISION. But you might need to test that manually...

**Comment by henriksunity** [comment]
No further comments than @skovhus.

File: src/client/components/PullRequestList/PullRequestListItem/index.js (hunk 1 above)
Comment (kkateq) [inline src/client/components/PullRequestList/PullRequestListItem/index.js]: Sure, we could handle, but right now only branch head revision. Will add this as a task for this.

File: src/universal/services/ono/queries/pullrequests/index.js (hunk 4 above)
Comment (kkateq) [inline src/universal/services/ono/queries/pullrequests/index.js]: The current implementation of target filter in GraphQL allows passing all of them, values are passing as strings and are not validated against graphene enum `PullRequestSourceType`. That's why it's not an error now. I'll add note to ono PR.

File: src/universal/services/ono/queries/pullrequests/index.js [hunk 5]
```diff
@@ -106,11 +108,11 @@ export type RepositoryGraphType = {
 }
 
 export const PullRequestSource = {
-  BRANCH: 'branch',
-  REV: 'rev',
+  BRANCH: 'BRANCH',
```
Comment (skovhus) [inline src/universal/services/ono/queries/pullrequests/index.js]: `TAG` and `BOOK` is missing.

File: src/universal/services/ono/queries/pullrequests/index.js [hunk 6]
```diff
@@ -112,18 +109,30 @@ export type RepositoryGraphType = {
   name: string
 }
 
+export const PullRequestSource = {
```
Comment (skovhus) [inline src/universal/services/ono/queries/pullrequests/index.js]: Not sure we we are going to use them, but `TAG` and `BOOK` are some different sources.

**Pull request merged**